src/provider/CoinbaseWalletProvider.ts

```diff
@@ -81,6 +82,8 @@ export class CoinbaseWalletProvider
 
   private hasMadeFirstChainChangedEmission = false
 
+  private _supportsAddressSwitching?: boolean
```

Comment:
the leading underscore seems redundant with TS's `private`

Reply:
7f6f2a59fd1ea9fa93842c9122a4da01609ff921